webapp/actions/admin_actions.jsx

```diff
@@ -2,201 +2,173 @@
 // See License.txt for license information.
 
 import Client from 'client/web_client.jsx';
-import * as AsyncClient from 'utils/async_client.jsx';
 import {browserHistory} from 'react-router/es6';
 
 // Redux actions
 import store from 'stores/redux_store.jsx';
 const dispatch = store.dispatch;
 const getState = store.getState;
 
-import {getUser} from 'mattermost-redux/actions/users';
+import {updateUserMfa, updateUserPassword} from 'mattermost-redux/actions/users';
+import {
```

Comment:
At some point, should we just go back to having this do `import * as AdminActions from ...`? :p